components/unit/file-explorer/UnitFileExplorer.vue

```diff
@@ -145,6 +151,45 @@ export default {
         bottom: `-${w - 18}px`,
         maxWidth: `${h - w - 20}px`
       }
+    },
+    nFiles() {
+      return this.fileManager.fileList.length
+    },
+    totalSize() {
+      return _.sumBy(this.fileManager.fileList, f => f.size)
+    },
+    dataSizeString() {
+      return humanReadableFileSize(this.totalSize)
+    },
+    fileExtensionsString() {
+      const extensions = this.fileManager.fileList
+        .map(f => f.name.match(/^.+\.(.+?)$/)?.[1])
+        .filter(m => !_.isUndefined(m))
+
+      const occurrences = _.mapValues(
+        _.groupBy(extensions, _.identity),
+        v => v.length
+      )
+      const orderedCounts = _.sortBy(
+        Object.entries(occurrences),
+        ([_ext, count]) => -count
+      )
+      const showAtMost = 2
+      const topExtensions = _.take(orderedCounts, showAtMost)
+      const descriptions = topExtensions.map(
+        ([ext, count]) => `${count} ${ext}${count === 1 ? '' : 's'}`
```

Comment:
I don't like this format: it's weird to see "3 jss" or "5 jsons", I'd rather write "3 .js" and "5 .json"